### ISSUE TITLE
[BUG] - New document breaks the timeline

### ISSUE THREAD

**Issue by feeef** [pencil2d/pencil#374]
After doing some work with Pencil, going to File -> New visually breaks the timeline.

**Comment by ManuAros** [comment]
I don't know if this belongs here but I also noticed that when you are drawing and you start over with a new file (with File -> New) you can't start drawing before you move focus to another layer and back to bitmap layer. I don't know if this is because it's the camera layer that is set as default layer when you choose FIle-> New.

**Comment by carlosgf** [comment]
Maybe I should close this #300 ?, which is the same as yours. Besides that it is best understood with your words.

(There is no need to keep duplicate issues) :)

**Comment by feeef** [comment]
Sorry for the duplicate!

**Comment by carlosgf** [comment]
No problem!

(I will close mine. You have the best vocabulary for explaining things)
:+1:

**Comment by tasaif** [comment]
Screenshot please and steps to duplicate bug ("doing some work" is vague)

@ManuAros that should be a separate issue

**Comment by feeef** [comment]
You're right @tasaif. This bug was primarily a bug reminder for myself :)

I haven't been able to reproduce this bug where the layout is broken. It was the keyframes from the previous project still being half visible in the timeline. I have been doing some work on the timeline display when implementing frames dragging so I may I fixed it.

The other thing was the camera layer being selected by default. It is not intuitive and looks broken. We should have the bitmap layer selected by default just like when the app starts up.

**Comment by feeef** [comment]
I close this bug as I can't reproduce it. @ghost should re-open his bug as it was describing the rest of the bug. sorry about that!
